Bot/modules/db.module.js

```diff
@@ -0,0 +1,272 @@
+'use strict';
+
+const UIDGenerator = require('uid-generator');
+const pool = require('../../DB/pool');
+const config = require('../../config');
```

Comment:
It will be better to use dependency injection to pass db connection poll and config here

Reply:
Have never used injections in JS but seems to be good, okay